extensions/binding/org.eclipse.smarthome.binding.mqttbroker/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,34 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MqttBroker Binding
+Bundle-SymbolicName: org.eclipse.smarthome.binding.mqttbroker;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: ., lib/moquette-broker-0.10.jar, lib/netty-all-4.1.13.Final.jar
+Import-Package: 
+ com.google.common.collect;version="10.0.1",
+ org.apache.commons.io;version="2.2.0",
+ org.apache.commons.lang,
+ org.apache.commons.net.util;version="3.2.0",
```

Comment:
"The manifest must not have any version constraint on package imports, unless this is thoughtfully added. Note that Eclipse automatically adds these constraints based on the version in the target platform, which might be too high in many cases."

So, if you added the version constraints because you know it does not work with a version lower then the given one, it is okay for me.
If it has only been added by the Eclipse IDE because this is the currently used version, I assume you have to remove them.